src/components/section/index.js

```diff
@@ -33,17 +33,25 @@ const styles = StyleSheet.create({
 /**
  * Component used to contain group of Blocks.
  */
-const Section = ({ children, style }, { contentDirection, containerStretch }) => (
-  <View
-    style={[
-      (contentDirection === 'vertical' ? styles.vertical : styles.horizontal),
-      (containerStretch ? styles.stretch : null),
-      style,
-    ]}
-  >
-    { children }
-  </View>
-);
+const Section = ({ children, style, stretch }, { contentDirection, containerStretch }) => {
+  if (!containerStretch && !!stretch) {
```

Comment:
Should we add `process.env.NODE_ENV` check, so that this code gets stripped from production build?

Reply:
It does, adding it now.